content/en/docs/concepts/storage/volumes.md

```diff
@@ -1381,6 +1381,10 @@ plugin path on each node and in some cases the control plane nodes as well.
 Pods interact with FlexVolume drivers through the `flexvolume` in-tree volume plugin.
 For more details, see the [FlexVolume](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-storage/flexvolume.md) examples.
 
+{{< note >}}
+Flexvolume is deprecated starting v1.23. Out-of-tree CSI driver is the recommended way to write volume drivers in Kubernetes. Maintainers of Flexvolume drivers should implement a CSI Driver and move users of Flexvolume to CSI. Users of Flexvolume should move their workloads to CSI Driver.
```

Comment:
@chetak123 The correct capitalization seems to be `FlexVolume` according to [this comment](https://github.com/kubernetes/sig-release/discussions/1709#discussioncomment-1515644) and also [this documentation for FlexVolume](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-storage/flexvolume.md)